style.css

```diff
@@ -272,6 +527,94 @@ textarea {
   height: auto;
 }
 
+@media (max-width: 768px) {
+  .geometry-6 {
+    display: none;
+  }
+
+  .geometry-7 {
+    display: none;
+  }
+
+  #message-label {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .started {
+    flex-direction: column;
+    background-color: rgba(249, 249, 250, 1);
+    background-image: url(./images/image-geometry_7.svg);
+    background-repeat: no-repeat;
+    background-position-x: right;
+    background-position-y: bottom;
+  }
+
+  #shapes-bottom {
+    display: none;
+  }
+
+  .geometry-6 {
+    position: absolute;
+  }
+
+  form {
+    width: 40%;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .title2 {
+    text-align: center;
+    margin-left: 0;
+    padding-left: 0%;
+  }
+
+  .presentation2 {
+    padding-left: 10%;
+    padding-right: 10%;
+    text-align: justify;
+  }
+
+  form > ul {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    gap: 5px 5px;
+    grid-template-areas:
+      ". ."
+      ". .";
+    padding-left: 5%;
+    height: 270px;
+  }
+
+  #t-area {
+    grid-column: span 2;
+    width: 105%;
+    padding-top: 0%;
+  }
+
+  label {
+    position: relative;
+    padding-left: 0%;
```

Comment:
- [ ] @emmanuelcortes2, you did a remarkable job on this project 👏. However, I recommend you remove the ` padding-left: 0%;` because it makes the `message` label move down.